examples/kubernetes/deployment.yaml

```diff
@@ -64,3 +72,5 @@ spec:
           value: "conf/"
         - name: "CONFIGMAP_LABELS"
           value: "app=k8s-sidecar-injector"
+        - name: "CONFIGMAP_NAMESPACE"
```

Comment:
This will work.

I was thinking (in order to provide a smoother out of the box experience) that we could change https://github.com/tumblr/k8s-sidecar-injector/blob/master/entrypoint.sh#L21 to omit the `--configmap-namespace` flag if `CONFIGMAP_NAMESPACE` is empty. This will use the namespace the injector is deployed into automatically, removing some config duplication and Doing The Right Thing ™️ in common cases.